README-REST.md

```diff
@@ -0,0 +1,1122 @@
+
+#### Cancel order
+```python
+response = bitvavo.cancelOrder('BTC-EUR', '5986db7b-8d6e-4577-8003-22f363fb3626')
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+{
+  "orderId": "5986db7b-8d6e-4577-8003-22f363fb3626"
+}
+```
+</details>
+
+#### Get orders
+Returns the same as get order, but can be used to return multiple orders at once.
+```python
+# options: limit, start, end, orderIdFrom, orderIdTo
+response = bitvavo.getOrders('BTC-EUR', {})
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+[
+  {
+    "orderId": "bad72641-7755-464c-8dcb-7c1d59b142ab",
+    "market": "BTC-EUR",
+    "created": 1548670024870,
+    "updated": 1548670024870,
+    "status": "partiallyFilled",
+    "side": "buy",
+    "orderType": "limit",
+    "amount": "1",
+    "amountRemaining": "0.5",
+    "price": "3000",
+    "onHold": "1504.5",
+    "onHoldCurrency": "EUR",
+    "filledAmount": "0.5",
+    "filledAmountQuote": "1500",
+    "feePaid": "3",
+    "feeCurrency": "EUR",
+    "fills": [
+      {
+        "id": "108c3633-0276-4480-a902-17a01829deae",
+        "timestamp": 1548671992530,
+        "amount": "0.5",
+        "price": "3000",
+        "taker": false,
+        "fee": "3",
+        "feeCurrency": "EUR",
+        "settled": true
+      }
+    ],
+    "selfTradePrevention": "decrementAndCancel",
+    "visible": true,
+    "timeInForce": "GTC",
+    "postOnly": false
+  },
+  {
+    "orderId": "da1d8330-d6b7-4753-800a-01ad510a679b",
+    "market": "BTC-EUR",
+    "created": 1548666570234,
+    "updated": 1548666570234,
+    "status": "filled",
+    "side": "sell",
+    "orderType": "limit",
+    "amount": "0.1",
+    "amountRemaining": "0",
+    "price": "4000",
+    "onHold": "0",
+    "onHoldCurrency": "BTC",
+    "filledAmount": "0.1",
+    "filledAmountQuote": "400",
+    "feePaid": "0.8",
+    "feeCurrency": "EUR",
+    "fills": [
+      {
+        "id": "79e4bf2f-4fac-4895-9bb2-a5c9c6e2ff3f",
+        "timestamp": 1548666712071,
+        "amount": "0.1",
+        "price": "4000",
+        "taker": false,
+        "fee": "0.8",
+        "feeCurrency": "EUR",
+        "settled": true
+      }
+    ],
+    "selfTradePrevention": "decrementAndCancel",
+    "visible": true,
+    "timeInForce": "GTC",
+    "postOnly": false
+  },
+  ...
+]
+```
+</details>
+
+#### Cancel orders
+Cancels all orders in a market. If no market is specified, all orders of an account will be canceled.
+```python
+# options: market
+response = bitvavo.cancelOrders({})
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+[
+  {
+    "orderId": "4f9a809b-859f-4d8d-97b3-037113cdf2d0"
+  }, 
+  {
+    "orderId": "95313ae5-ad65-4430-a0fb-63591bbc337c".
+  }, 
+  {
+    "orderId": "2465c3ab-5ae2-4d4d-bec7-345f51b3494d"
+  },
+  ...
+]
+```
+</details>
+
+#### Get orders open
+Returns all orders which are not filled or canceled.
+```python
+# options: market
+response = bitvavo.ordersOpen({})
```

Comment:
A lot of the commands above should probably have a working example like this one:
```suggestion
response = bitvavo.ordersOpen({ market: "BTC-EUR" })
```

Reply:
Will update the API ref first, then either backport here or point to the API ref.